### PR TITLE
Geant4CaloHit fix and Jenkinsfile 

### PR DESCRIPTION
**Pull request by vvolkl**
The data type used in the  SimpleCaloSD sensitive detector was  planned to be removed in dd4hep, so I switch to a fcc type. This should be completely analogous to the tracker where it was done already a while ago.  

Also I save the trackID of CaloHits in the currently unused bits field, which is useful for the CaloHitMatching I am doing.

Includes my Jenkinsfile because I forgot to remove it and  should anyway be used by the CI very soon.

### CODE REVIEW

**Comment by vvolkl** [comment]
@phsft-bot build

File: Detector/DetSensitive/src/AggregateCalorimeterSD.cpp [hunk 1]
```diff
@@ -45,11 +45,11 @@ bool AggregateCalorimeterSD::ProcessHits(G4Step* aStep, G4TouchableHistory*) {
   dd4hep::Position pos(midPos.x(), midPos.y(), midPos.z());
```
Comment (JavierCVilla) [inline Detector/DetSensitive/src/AggregateCalorimeterSD.cpp]: Is `pos` still needed?

File: Detector/DetSensitive/src/AggregateCalorimeterSD.cpp [hunk 2]
```diff
@@ -58,9 +58,14 @@ bool AggregateCalorimeterSD::ProcessHits(G4Step* aStep, G4TouchableHistory*) {
...
   // if not, create a new hit
   // deleted in ~G4Event
-  hitMatch = new dd4hep::sim::Geant4CalorimeterHit(pos);
+  hitMatch = new fcc::Geant4CaloHit(0, // track->GetTrackID()
+                                    0, // track->GetDefinition()->GetPDGEncoding()
+                                    edep,
+                                    0 // track ->GetGlobalTime()
+                                    ) ;
+
+  hit->position = midPos;
```
Comment (JavierCVilla) [inline Detector/DetSensitive/src/AggregateCalorimeterSD.cpp]: Is this `hit` related to `hitMatch` at this point?

Comment (vvolkl) [reply Detector/DetSensitive/src/AggregateCalorimeterSD.cpp]: good catch, this should be hitMatch

File: Detector/DetSensitive/src/AggregateCalorimeterSD.cpp (hunk 1 above)
Comment (vvolkl) [reply Detector/DetSensitive/src/AggregateCalorimeterSD.cpp]: no

**Pull request merged**